src/panel/manager.vala

```diff
@@ -612,12 +601,6 @@ namespace Budgie {
 				return;
 			}
 
-			/* Manual configuration from user met the expected migration path. Proceed as normal. */
-			if (panel_plugin_manager.migrate_load_requirements_met) {
-				message("Budgie Migration skipped due to user meeting migration requirements");
-				return;
-			}
-
 			message("Budgie Migration initiated");
```

Comment:
Can we remove all of the migration code now?  This was for a very old version of budgie-desktop